### DISCUSSION TITLE
Hotspot doesn't show up after install on Commodore CVR-200

### DISCUSSION THREAD

**Discussion by skruehoetl** [Hypfer/Valetudo#1954] [Q&A / Support] [answered]
Sorry for asking stupid questions - but I ran into an issue on a fresh install of Valetudo on my Commodore CVR-200:

What I did initially:
- I did the install als per the [Readme.md
](https://github.com/Hypfer/valetudo-crl200s-root)
- The dustbuild/er firmware I'm using is for the viomi.vacuum.v6
- I rebooted
- I did the "/opt/8821cs/enable_8821cs.sh" step as described and rebooted again

The problem:
- There is no hotspot looking remotely like valetudo (the Android companion app also doesn't find one.

What I did then:
- I disabled/re-enabled the 8821cs module (rebooting in between) - no effect
- I checked for the modules with "lsmod" on adb shell and found "8189es  2023555  0" (with the 8189es and the renamed 8821cs module - except with different sizes (2023555 vs 1542814))
- I looked (stupidly) at the ifconfig output showing me wlan0 uninitialized...
- I tried repeating the install - but that of course didn't work

... please help?

**Comment by Hypfer** [comment, marked as answer]
Did you also `press and hold the two buttons until the robot informs you that Wi-Fi has been reset.` as mentioned in the last step of the rooting instructions?

**Reply by skruehoetl** [reply]
Ahm... That did it. Thanks. I must have skipped those lines (which is more than a bit embarrassing). Anyway - huge thanks for helping!
